### DISCUSSION TITLE
Supabase Self Host

### DISCUSSION THREAD

**Discussion by Mohamed-Emara77** [supabase/supabase#2070] [Questions] [answered]
I am a little new to hosting world so I would like to know if it is possible to host Supabase on a shared host via Cpanel =>[ Setup Nodejs app]  which has a postgresql database hosted .
because I only see CLI install which requires root access ( if I do understand correctly this will mean that you need a VPS not a shared host ) .

**Comment by inian** [comment, marked as answer]
Hi @Mohamed-Emara77, the root access requirement is because of the dependency on Docker right? 

Since we string together a lot of different tools, each with their own toolchains and languages, it will be hard to provide a good experience for local dev without something like Docker. So, if you are not able to install Docker on the VPS, I don't think you can use the cli either.

**Reply by Mohamed-Emara77** [reply]
Okay bro thanks for your time and effort.
